model/src/main/proto/javaclasses/mealorder/c/commands.proto

```diff
@@ -56,68 +56,56 @@ message AddVendor {
     // Phone numbers of the vendor.
     repeated PhoneNumber phone_numbers = 4;
 
-    // A daily deadline.
+    // Time of closing orders for editing.
+    // Orders have to be added to purchase order.
```

Comment:
_// Time of closing orders for this vendor for editing._
_// Purchase order for this vendor will be created exactly at this time._